projects/schematics/src/migrations/4_0/missing-packages/missing-packages.ts

```diff
@@ -0,0 +1,46 @@
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import {
+  migrateMissingPackage,
+  MissingPackageMigration,
+} from '../../mechanism/missing-packages/missing-packages';
+
+const MISSING_PACKAGE_DATA: MissingPackageMigration[] = [
+  {
+    package: '@spartacus/checkout',
+    comment:
+      "We've found imports from @spartacus/checkout package which is not installed. If you are using the checkout feature you can configure it by running schematics `ng add @spartacus/checkout`. If you only need to install package add it with npm (`npm i @spartacus/checkout`) or yarn (`yarn add @spartacus/checkout`). If you are not using checkout package check why you have imports from this library.",
```

Comment:
These comments all look alike. If they're not specified, we can generate them based on the `package`.